uninstall.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# Installation script for cake-autorate.sh
```

Comment:
```suggestion
# Uninstall script for cake-autorate.sh
```

Reply:
Could we make the same change for setup.sh, i.e. "Installation script for cake-autorate"?